auto_submit/Dockerfile

```diff
@@ -4,7 +4,7 @@
 
 # Use Google's official Dart image.
 # https://hub.docker.com/r/google/dart-runtime/
-FROM google/dart-runtime
+FROM google/dart-runtime@sha256:20efd786c6acd80aac80da76d26fed0a5a96523ca6f1bd0d6ff43ea394f746ca
```

Comment:
Any reason we use different versions for these two apps?

Reply:
I'm not sure the reason for the difference here, but we could use this change to sync to use the same image if desired.

Reply:
Let's also use dev here.